blog/releases/announcing-knative-v0-17-release.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Version v0.17 release"
+title: "Version 0.17 release"
```

Comment:
According to https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0 we're using v0.x now instead